cogs/vxtwitconverter/eventsCore.py

```diff
@@ -0,0 +1,42 @@
+from discord import Message, channel
+from urlextract import URLExtract
+
+from cogs.vxtwitconverter.core import Core
```

Comment:
Cog doesn't load, need the following changes:
```suggestion
from .core import Core
```